atom/browser/native_window_mac.mm

```diff
@@ -754,6 +757,16 @@ static bool FromV8(v8::Isolate* isolate, v8::Handle<v8::Value> val,
     [window_ setOpaque:NO];
   }
 
+  if (base::mac::IsAtLeastOS10_12()) {
+    // Create a tab only if tabbing identifier is specified and window has
+    // a native title bar.
+    if (tabbing_identifier_.empty() || transparent() || !has_frame()) {
+      [window_ setTabbingMode:NSWindowTabbingModeDisallowed];
+    } else {
+      [window_ setTabbingIdentifier:base::SysUTF8ToNSString(tabbing_identifier_)];
```

Comment:
Same with this one, it is 10.12, https://developer.apple.com/reference/appkit/nswindow/1644704-tabbingidentifier?language=objc